lib/msal-core/src/UserAgentApplication.ts

```diff
@@ -345,6 +354,31 @@ export class UserAgentApplication {
     }
   }
 
+  //#region Redirect Callbacks
+  /**
+   * Sets the callback functions for the redirect flow to send back the success or error object.
+   * @param {tokenReceivedCallback} successCallback - Callback which contains the AuthResponse object, containing data from the server.
+   * @param {errorReceivedCallback} errorCallback - Callback which contains a AuthError object, containing error data from either the server
+   * or the library, depending on the origin of the error.
+   */
+  setRedirectCallbacks(successCallback: tokenReceivedCallback, errorCallback: errorReceivedCallback): void {
```

Comment:
2 questions, if someone tries to make a call without setting these will we throw? 
Does the old api still work if they pass in that way ?